backend/models/request_user.py

```diff
@@ -101,6 +102,22 @@ def send_notification(self, current_institution):
             entity_type=entity_type
         )
 
+    """ 
+    The sender, in this case, is the user who is asking to be an institution's members.
+    The user is who answered the request.
+    The institution_key to which the sender requested to be member.
+    Entity_type can be REJECTED_LINK or ACCEPTED_LINK.
+    """
+    def send_response_notification(self, user, institution_key, entity_type):
+        """Method sends response notification to sender request."""
+        send_message_notification(
```

Comment:
Check the possibility of generalizing this function. In the superclass invite, there already exists a 'send_response_notification' method, it always sends the invite key in the entity_key field, but could get to receive the entity_key as an optional parameter and if it isn't passed, use the invite key. This way it is easier to resolve the issue with the other invites that are also having problems with the notification.